src/TTrigger.cpp

```diff
@@ -1137,7 +1137,6 @@ bool TTrigger::match(char* subject, const QString& toMatch, int line, int posOff
 }
 
 
-// Die Musternummer wird ID im color-pattern lookup table
```

Comment:
This is just irreplacably deleted?

Reply:
It's an incomplete sentence which makes no sense and a complete English comment is below that.

Reply:
This also seems to stem back 11y and was outdated by the last refactoring in #1840, so all is good.